requirements.txt

```diff
@@ -53,11 +53,12 @@ django-grappelli==2.12.3  # via -r requirements.in
 django-inlinecss==0.3.0   # via -r requirements.in
 django-libsass==0.7       # via -r requirements.in
 django-redis==4.10.0      # via -r requirements.in
+django-rest-passwordreset==1.1.0  # via -r requirements.in
```

Comment:
@Agrendalath It looks like this repo is getting abandoned, with no commits for the past 3 months.
Do you think it's possible to replace it with one of the DRF recommended libraries from https://www.django-rest-framework.org/api-guide/authentication/#django-rest-auth?

Reply:
@giovannicimolin, the quick grep shows two candidates:
- [Tivix/django-rest-auth](https://github.com/Tivix/django-rest-auth) - this one is abandoned and we had to make [a fork of this](https://github.com/open-craft/django-rest-auth) to support SimpleJWT in Sprints.
-  [sunscrapers/djoser](https://github.com/sunscrapers/djoser), which [doesn't have](https://djoser.readthedocs.io/en/latest/base_endpoints.html#reset-password-confirmation) an endpoint to verify tokens.

`django-rest-passwordreset` actually supports Django==3.0.0 (cf. https://github.com/anexia-it/django-rest-passwordreset/issues/78), but, indeed, it doesn't seem to be maintained. However I haven't found a better option that would support our use-case in the expected timebox.

Reply:
Let's use this one for now then.